v1.0/coordinateSpace.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/CoordinateSpace",
+  "description": "Structured information describing a coordinate space.",
+  "required": [
+    "alias",
+    "anatomicalOrientation",
+    "fullName",
+    "origin",
+    "releaseDate",
+    "unit",
+    "versionIdentifier"
+  ],
+  "properties": {
+    "alias": {
+      "type": "string",
+      "_instruction": "Enter the user-defined identifier for this coordinate space (e.g. a short name)."
+    },
+    "anatomicalOrientation": {
+      "_instruction": "Add the axes orientation denoted in standard anatomical terms of direction (in 2D or 3D).",
```

Comment:
"Add the axes orientation denoted in standard anatomical terms of direction (stated as XYZ)."

Reply:
Suggestion based on yours: 
"Add the axes orientation denoted in standard anatomical terms of direction (stated as XYZ for 3D and XY for 2D)."

For the 2D orientations there is no Z, right?

Reply:
Always 3D.